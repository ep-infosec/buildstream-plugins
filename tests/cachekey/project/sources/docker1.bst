kind: import
sources:
- kind: docker
  image: theimage
  ref: thisistheimagedigest
