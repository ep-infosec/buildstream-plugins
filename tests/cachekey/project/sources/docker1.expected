917564bc9f8a95493ff09c8426fad5eb3bf72f36388830c7a9680e31105126f2